features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -651,7 +651,7 @@
   "console.ui.app_title": "Console | WSO2 Identity Server",
   "console.ui.app_name": "Console",
   "console.ui.app_logo_path": "/assets/images/logo.svg",
-  "console.ui.product_name": "Identity Server",
+  "console.ui.product_name": "WSO2 Identity Server",
```

Comment:
If we use wouldn't it be redundant with the wso2 logo?

If possible can u post a screen grab with this change

Reply:
We removed because when the WSO2 `product logo` is present and the prefix `WSO2` is redundant if not repeated.

<img width="384" alt="sample" src="https://user-images.githubusercontent.com/25561152/108870084-cec3ce00-761d-11eb-87ab-9f1156c9c908.png">

Reply:
This **will not be used in product logo area**. It is redundant only if we use it in the product logo area. But this property serves a different purpose. We use it for the `footer` and other `hint/explanation` related texts.

<img width="390" alt="Screenshot 2021-02-23 at 21 30 36" src="https://user-images.githubusercontent.com/25561152/108871998-be145780-761f-11eb-9a2e-92abd93177c1.png">

<img width="748" alt="Screenshot 2021-02-23 at 21 37 54" src="https://user-images.githubusercontent.com/25561152/108872010-bfde1b00-761f-11eb-8630-8f05dad3132a.png">

https://github.com/wso2/carbon-identity-framework/blob/dd37cf3f588f3c4c7f1b157d7d8106bd47bd93bc/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json#L651